cmd/operator/main.go

```diff
@@ -266,11 +267,28 @@ func Main() int {
 		return 1
 	}
 
-	pao, err := prometheusagentcontroller.New(ctx, cfg, log.With(logger, "component", "prometheusagentoperator"), r)
-	if err != nil {
-		fmt.Fprint(os.Stderr, "instantiating prometheus-agent controller failed: ", err)
-		cancel()
-		return 1
+	var pao *prometheusagentcontroller.Operator
+	verbs := map[string][]string{
+		monitoringv1alpha1.PrometheusAgentName:                           {"get", "list", "watch"},
+		fmt.Sprintf("%s/status", monitoringv1alpha1.PrometheusAgentName): {"update"},
+	}
```

Comment:
nit: part of me feels like these might belong to either `monitoringv1alpha1` or `prometheusagentcontroller` inside a function like `GetPrometheusAgentVerbs` but I'll let @simonpasquier give his 2 cents, this might just be me being picky

Reply:
I see your point. I think we can leave it for now and see later (as we reuse the same pattern) if it makes sense to move it back.